File: WebSource/WebSource/Models/CSale.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebSource.Models
{
    public class CSale
    {
        public int? shop_id { get; set; }
        public int sale_id { get; set; }
        public CProduct cproduct { get; set; }
        public CUser agent { get; set; }
        public int? prod_qty { get; set; }
        public DateTime saleDate { get; set; }
        public TimeSpan saleTime { get; set; }
        public String cust_name { get; set; }
        public String cust_phone { get; set; }

        public CSale(sale sale, CProduct product, CUser user)
        {
            shop_id = sale.shop_id;
            sale_id = sale.sale_id;
            cproduct = product;
            agent = user;
            prod_qty = sale.prod_quant;
            saleDate = sale.sale_date;
            saleTime = sale.sale_time;
            cust_name = sale.cust_name;
            cust_phone = sale.cust_phone;

        }
    }
}

File: androidApp/POS/app/src/main/java/com/apptriangle/pos/model/CSale.java
package com.apptriangle.pos.model;

import java.sql.Timestamp;
import java.util.ArrayList;

/**
 * Created by zeeshan on 5/3/2018.
 */
public class CSale {
    public int shop_id ;
    public int sale_id ;
    public ArrayList<Product> cproduct ;
    public User agent ;
    public int prod_qty ;
    public String saleDate ;
    public String saleTime ;
    public String cust_name ;
    public String cust_phone ;
    public Double amount_paid, total_amount;

    public CSale(Sale s){
        this.shop_id = s.shop_id;
        this.sale_id = s.sale_id;
        this.cproduct = new ArrayList<>();
        this.cproduct.add(s.cproduct);
        this.agent = s.agent;
        this.prod_qty = s.prod_qty;
        this.saleDate = s.saleDate;
        this.saleTime = s.saleTime;
        this.cust_name = s.cust_name;
        this.cust_phone = s.cust_phone;
    }
}
